### PR TITLE
Feature/guble #90 healthcheck for kvstore

### PR DESCRIPTION
**Pull request by marian-craciunescu**

### CODE REVIEW

File: server/router.go [hunk 1]
```diff
@@ -115,6 +115,12 @@ func (router *router) Check() error {
 	if err != nil {
 		return err
 	}
+
+	err = router.kvStore.Check()
```
Comment (cosminrentea) [inline server/router.go]: check if kvStore is null first

File: store/memory.go [hunk 2]
```diff
@@ -92,3 +92,8 @@ func (kvStore *MemoryKVStore) getSchema(schema string) map[string][]byte {
 	kvStore.data[schema] = s
 	return s
 }
+
+func (kvStore *MemoryKVStore) Check() error {
+	// if the memory is full other parts of guble will crash so no check is here
```
Comment (cosminrentea) [inline store/memory.go]: remove comment

File: store/sqlite.go [hunk 3]
```diff
@@ -163,6 +163,16 @@ func (kvStore *SqliteKVStore) Open() error {
 	return nil
 }
 
+func (kvStore *SqliteKVStore) Check() error {
+
+	if err := kvStore.db.DB().Ping(); err != nil {
```
Comment (cosminrentea) [inline store/sqlite.go]: check kvStore db is not nil

File: server/router.go (hunk 1 above)
Comment (marian-craciunescu) [inline server/router.go]: done

**Pull request merged**